.github/workflows/ghost-cli.yml

```diff
@@ -0,0 +1,20 @@
+name: Ghost-CLI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '10'
+      - run: npm install -g ghost-cli@latest
+      - run: zip -r ghost.zip .
+      - run: mkdir test
+      - run: ghost install local --zip ../ghost.zip
+        working-directory: test
```

Comment:
Fun fact: running `ghost install -d test` will install ghost in the `test` folder 👻

Reply:
Oh nice! I'll fix that with my next incoming PR

Reply:
I just saw that! You can run any command with the `-d` flag 😃